<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Map</class>
 <widget class="QWidget" name="Map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>775</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_map"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string> Дистанция</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Модуль 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="module1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Модуль 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="module2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Модуль 3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="module3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Модуль 3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="module5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="xx">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="yy">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Проверка подключения</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Модуль 0</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="prov0">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Модуль 1</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="prov1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Модуль 2</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="prov2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Модуль 3</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="prov3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Модуль 4</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QLabel" name="prov4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="prov5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="prov6">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="prov7">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Error</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLabel" name="error">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>error_code1</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QLabel" name="error_code1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>error_code2</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLabel" name="error_code2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>error_code3</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="error_code3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>error_code4</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QLabel" name="error_code4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>калибровка</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
